sphinx_js/typedoc.py

```diff
@@ -153,9 +153,15 @@ def make_type_name(self, type):
 
     def make_type(self, type):
         """Construct a jsdoc type entry"""
-        return {
-            'names': [self.make_type_name(type)]
-        }
+        type_name = self.make_type_name(type)
+        if isinstance(type_name, list):
+            return {
+                'names': type_name
+            }
+        else:
+            return {
+                'names': [type_name]
+            }
```

Comment:
Same as previous comment, if we refactor make_type_name this wouldn't be necessary, would it?

Reply:
Should be done with commit 31c4aba, please have a look again.